kuma/wiki/tests/test_views_list.py

```diff
@@ -121,3 +145,201 @@ def test_revisions_request_invalid_pages(root_doc, client):
+    assert 'text/html' in resp['Content-Type']
+    assert len(dom.find('.document-list li')) == len(exp_docs)
+    selector = 'ul.document-list li a[href="/{}/docs/{}"]'
+    for doc in exp_docs:
+        assert len(dom.find(selector.format(doc.locale, doc.slug))) == 1
+
+
+@pytest.mark.parametrize('locale', ['en-US', 'de', 'fr'])
+def test_list_top_level(redirect_doc, doc_hierarchy_with_zones, client,
+                        locale):
+    if locale == 'en-US':
+        exp_docs = (doc_hierarchy_with_zones.top,)
+    else:
+        exp_docs = (doc_hierarchy_with_zones.top.translated_to(locale),)
+
+    url = reverse('wiki.top_level', locale=locale)
+    resp = client.get(url)
+    dom = pq(resp.content)
+    assert resp.status_code == 200
+    assert 'public' in resp['Cache-Control']
+    assert 's-maxage' in resp['Cache-Control']
+    assert 'text/html' in resp['Content-Type']
+    assert len(dom.find('.document-list li')) == len(exp_docs)
+    selector = 'ul.document-list li a[href="/{}/docs/{}"]'
+    for doc in exp_docs:
+        assert len(dom.find(selector.format(doc.locale, doc.slug))) == 1
+
+
+@pytest.mark.parametrize('tag', ['foo', 'bar'])
+@pytest.mark.parametrize('locale', ['en-US', 'de', 'fr'])
+def test_list_with_localization_tag(redirect_doc, doc_hierarchy_with_zones,
+                                    client, locale, tag):
+    top_doc = doc_hierarchy_with_zones.top
+    bottom_doc = doc_hierarchy_with_zones.bottom
+    de_doc = top_doc.translated_to('de')
+    for doc in (top_doc, bottom_doc, de_doc, redirect_doc):
+        doc.current_revision.localization_tags.set('foo', 'bar')
+    middle_bottom_doc = doc_hierarchy_with_zones.middle_bottom
+    middle_bottom_doc.current_revision.localization_tags.set('foobar')
+
+    if locale == 'en-US':
+        exp_docs = (top_doc, bottom_doc)
+    elif locale == 'de':
+        exp_docs = (de_doc,)
+    else:  # fr
+        exp_docs = ()
+
+    url = reverse('wiki.list_with_localization_tag', locale=locale,
+                  kwargs={'tag': tag})
+    resp = client.get(url)
+    dom = pq(resp.content)
+    assert resp.status_code == 200
+    assert 'public' in resp['Cache-Control']
+    assert 's-maxage' in resp['Cache-Control']
+    assert 'text/html' in resp['Content-Type']
+    assert len(dom.find('.document-list li')) == len(exp_docs)
+    selector = 'ul.document-list li a[href="/{}/docs/{}"]'
+    for doc in exp_docs:
+        assert len(dom.find(selector.format(doc.locale, doc.slug))) == 1
+
+
+@pytest.mark.parametrize('locale', ['en-US', 'de', 'fr'])
+def test_list_with_localization_tags(redirect_doc, doc_hierarchy_with_zones,
+                                     client, locale):
+    top_doc = doc_hierarchy_with_zones.top
+    bottom_doc = doc_hierarchy_with_zones.bottom
+    de_doc = top_doc.translated_to('de')
+    for doc in (top_doc, bottom_doc, de_doc, redirect_doc):
+        doc.current_revision.localization_tags.set('foo', 'bar')
```

Comment:
Nit: Similar issue - the test code implies that localization tags other that "inprogress" would be used.